scripts/jira_tickets/create_datacheck_tickets.pl

```diff
@@ -81,47 +90,38 @@ =head1 OPTIONS
 # Get file absolute path and basename
 my $dc_abs_path = abs_path($dc_file);
 my $dc_basename = fileparse($dc_abs_path, qr{\.[a-zA-Z0-9_]+$});
-# Get timestamp that will be included in the summary of each JIRA ticket
-my $timestamp = strftime("%d-%m-%Y %H:%M:%S", localtime time);
 # Get a new Utils::JIRA object to create the tickets for the given division and
 # release
 my $jira_adaptor = Bio::EnsEMBL::Compara::Utils::JIRA->new(-DIVISION => $division, -RELEASE => $release);
 # Parse Datacheck information from input TAP file
-my $testcase_failures = parse_datachecks($dc_file, $timestamp);
-# Create initial ticket for datacheck run - failures will become subtasks of this
-my $blocked_ticket_key = find_handover_ticket($jira_adaptor);
-my $dc_task_json_ticket = [{
-    assignee    => $jira_adaptor->{_user},
-    summary     => "$dc_basename ($timestamp)",
-    description => "Datacheck failures raised on $timestamp\nFrom file: {{$dc_abs_path}}",
-}];
-# Create subtask tickets for each datacheck subtest failure
-my @json_subtasks;
+my $testcase_failures = parse_datachecks($dc_file);
+# Create a task ticket for each datacheck subtest failure
+my @json_tasks;
 foreach my $testcase ( keys %$testcase_failures ) {
-    my $failure_subtask_json = {
-        summary     => $testcase,
+    my $failure_task_json = {
+        summary     => "Datacheck $testcase failed",
         description => $testcase_failures->{$testcase},
     };
-    push(@json_subtasks, $failure_subtask_json);
+    push(@json_tasks, $failure_task_json);
 }
-# Add subtasks to the initial ticket
-$dc_task_json_ticket->[0]->{subtasks} = \@json_subtasks;
 my $components = ['Datachecks', 'Production tasks'];
 my $categories = ['Bug::Internal', 'Production::Tasks'];
 # Create all JIRA tickets
 my $dc_task_keys = $jira_adaptor->create_tickets(
-    -JSON_OBJ         => $dc_task_json_ticket,
+    -JSON_OBJ         => \@json_tasks,
     -DEFAULT_PRIORITY => 'Blocker',
     -EXTRA_COMPONENTS => $components,
     -EXTRA_CATEGORIES => $categories,
+    -UPDATE           => $update,
     -DRY_RUN          => $dry_run
 );
 # Create a blocker issue link between the newly created datacheck ticket and the
 # handover ticket
+my $blocked_ticket_key = find_handover_ticket($jira_adaptor);
 $jira_adaptor->link_tickets('Blocks', $dc_task_keys->[0], $blocked_ticket_key, $dry_run);
```

Comment:
The "blocker link" is created only the first time the DC fails for that release, yes. But since every time this DC fails the ticket is reopened, the subtask `Handover of release DB` will still display an unresolved "issue blocked by" task.